api/src/feeds/impl/datasets_api_impl.py

```diff
@@ -39,6 +47,38 @@ def create_dataset_query():
             .join(Feed, Feed.id == Gtfsdataset.feed_id)
         )
 
+    @staticmethod
+    def _load_validation_report():
```

Comment:
Let's add unit tests that cover this function.

Reply:
this method is covered by the unit tests for the validation report